staging/src/k8s.io/client-go/copy.sh

```diff
@@ -131,7 +131,7 @@ function mvfolder {
     # the first rule is to convert import lines like `restclient "k8s.io/client-go/pkg/client/restclient"`,
     # where a package alias is the same the package name.
     find "${CLIENT_REPO}" -type f -name "*.go" -print0 | \
-        xargs -0 sed -i "s,${src_package} \"${CLIENT_REPO_FROM_SRC}/${src},${dst_package} \"${CLIENT_REPO_FROM_SRC}/${dst},g"
+        xargs -0 sed -i "s,\<${src_package} \"${CLIENT_REPO_FROM_SRC}/${src},${dst_package} \"${CLIENT_REPO_FROM_SRC}/${dst},g"
```

Comment:
It's to avoid rewriting such import:
```
import (
fcache "k8s.io/client-go/pkg/client/cache"
)
```
to
```
import (
fcore "k8s.io/client-go/core"
)
```
because when rewriting invocations, the script search for whole word, it does not rewrite the invocation from `fcache` to `fcore`.